x-pack/plugins/fleet/public/applications/fleet/components/index.ts

```diff
@@ -8,4 +8,4 @@
 export * from '../../../components';
 
 export * from './enrollment_instructions';
-export * from './search_bar';
+export * from './search_bar';
```

Comment:
This needs to be corrected

Reply:
I modified the line. Could you check again, please.